edx_toggles/toggles/__future__.py

```diff
@@ -1,8 +1,15 @@
 """
-Expose new-style waffle classes. In the future, these imports will be moved to edx_toggles.toggles. Applications that
-want to migrate to the new-style API can do so right now by importing from this module.
+This module had been created to expose new-style waffle classes. These are now available from edx_toggles.toggles.
 """
-# pylint: disable=unused-import
-from .internal.setting_toggle import SettingDictToggle, SettingToggle
-from .internal.waffle.flag import NonNamespacedWaffleFlag, WaffleFlag
-from .internal.waffle.switch import NonNamespacedWaffleSwitch, WaffleSwitch
+import warnings
+
+from . import *  # pylint: disable=unused-import,wildcard-import
```

Comment:
Can we just import the 4 classes explicitly?  Why are we changing this to a * import?  Do the imports need to change at all?

Reply:
I think that the `import *` makes it clearer which module needs to be imported.

Reply:
Maybe I’m missing something, but doesn’t this make the Legacy classes available from futures? Seems odd to me. In any case, this is temporary and it seems like you’ll be relying on search/replace when futures goes away, so whatever makes you happy. 🙂

Reply:
@regisb: I'll assume this continues to make you happy and my comments don't change anything, so feel free to mark resolved.

Reply:
:ok_hand: